keymanapp/keyman
# chore(common): Add Crowdin strings for Portuguese (pt-PT)

This adds the crowdin strings for [Portuguese, Portugal](https://translate.keyman.com/project/keyman/pt-PT), which is separate from [Portuguese, Brazilian](https://translate.keyman.com/project/keyman/pt-BR).

TODO before testing
- [x] @sgschantz to run the iOS and macOS projects through XCode to add pt-PT locale.

## User Testing
Note, the overall crowdin progress on Portuguese is 98%, so there will still be a few untranslated strings that show in English.

* **TEST_ANDROID**
1. Load the PR build of Keyman for Android
2. From the Keyman "Settings" menu, change the "Display Language" to "Português do Portugal"
3. Return to the Keyman app and verify the strings are in Portuguese

* **TEST_IOS**
1. Load the PR build of Keyman for Android
2. Set the device locale to Portuguese (does iOS distinguish Portugal from Brazil?)
3. Return to Keyman and verify strings are in Portuguese

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the device locale to Portuguese
3. Return to Keyman and verify strings are in Portuguese. Note, the macOS translation is 83% complete, so there will be a few English strings
